cobalt/storage/store/BUILD.gn

```diff
@@ -40,6 +40,7 @@ target(gtest_target_type, "memory_store_test") {
     "//cobalt/base",
     "//cobalt/storage:storage_constants",
     "//cobalt/test:run_all_unittests",
+    "//net:http_server",
```

Comment:
Again i'm surprised this is needed - which symbol / code pulls this in ?

Reply:
memory_store the target above , pulls in net , if i change that dependcy to only http_server it screams about needing other things in net